SECURITY.md

```diff
@@ -0,0 +1,21 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+***DO NOT CREATE AN ISSUE*** to report a vulnerability.
```

Comment:
I thought at some point there was the functionality to create private issues for these, but I don't see it now. Was it removed or did I imagine it? 🤔

Reply:
Nope, [Github doesn't have this feature](https://github.com/go-gitea/gitea/issues/3217), but [GitLab does](https://docs.gitlab.com/ee/user/project/issues/confidential_issues.html).

Reply:
Does this not depend on the security of the vulnerability?  HLJS allows most issues (include some security) to be reported via issues but if someone is concerned it's a high risk vulnerability then they can drop down to email.

Reply:
So HighlightJS does a mix of full disclosure and responsible disclosure? This does sound like a good strategy.

> they can drop down to email.

But we don't have an email yet and private issues would solve that issue (no pun intended).

Reply:
https://github.com/highlightjs/highlight.js/blob/main/SECURITY.md